docs/explanations.rst

```diff
@@ -152,18 +152,49 @@ E.g.::
                     break
 
 
-Trace/debug messages
-~~~~~~~~~~~~~~~~~~~~
+Verbose messages
+~~~~~~~~~~~~~~~~
+
+Useful to provide more information to the user that shouldn't be exposed when in brief mode for clarity and simplicity.
```

Comment:
Nitpick: in documentation, I would say: "Verbose messages are useful to..."

Reply:
Improved! Thanks!